.github/workflows/wheels.yml

```diff
@@ -153,6 +163,7 @@ jobs:
         env:
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}
+          CIBW_BUILD_FRONTEND: ${{ matrix.cibw_build_frontend || 'pip' }}

```

Comment:
Looks like the wasm build needs to specify `cibw_build_frontend`as something based on the failing build

Reply:
Hmm, according to the [docs](https://cibuildwheel.pypa.io/en/stable/options/#build-frontend) pyodide should ignore this setting, since only "build" is supported.